app/views/payouts/list.njk

```diff
@@ -83,12 +82,39 @@ Payments to your bank account - GOV.UK Pay
       </tbody>
     </table>
   {% else %}
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-    {% if payoutsReleaseDate %}
-    <p class="govuk-body">No payments to your bank account found on or after {{ payoutsReleaseDate.format('D MMMM YYYY') }}.</p>
-    <p class="govuk-body">For historic payments to your bank account please <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact support.</a></p>
+    {% if filterLiveAccounts %}
+      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+      {% if payoutsReleaseDate %}
+        <p class="govuk-body">No payments to your bank account found on or after {{ payoutsReleaseDate.format('D MMMM YYYY') }}.</p>
+        <p class="govuk-body">For historic payments to your bank account please <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact support.</a></p>
+      {% else %}
+        <p class="govuk-body">No payments to your bank account found.</p>
+      {% endif %}
     {% else %}
-    <p class="govuk-body">No payments to your bank account found.</p>
+      <h2 class="govuk-heading-m">Sample Report</h2>
+
+      <table class="govuk-table" id="payout-list">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Service</th>
+            <th scope="col" class="govuk-table__header govuk-table__header--numeric">Amount</th>
+            <th scope="col" class="govuk-table__header">View transactions</th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <td scope="row" class="govuk-table__cell">Test Service</td>
+            <td class="govuk-table__cell govuk-table__cell--numeric">
+              {{ 10000 | penceToPoundsWithCurrency }}
```

Comment:
We could also use `&#163;100.00` as this is static, but fine with ^

Reply:
Sure.